packages/extractor/src/extractor/mod.js

```diff
@@ -1,16 +1,35 @@
-// FIXME:
-// Find a better workaround.
-// Current issue: https://github.com/vitest-dev/vitest/issues/6953
-// We want to remove it, for the cross-runtime compatibility.
-import module from "node:module";
+import { IS_NODE_LIKE } from "../util.js";
+
+/**
+ * @internal
+ * Get the file path to the module, with vfs environments in mind.
+ * @param {string} specifier
+ * @returns {URL}
+ */
+function get_module_url(specifier) {
+	if (!IS_NODE_LIKE) {
```

Comment:
@xeho91 I believe it would be better to distinguish based on Node compatibility rather than simply on whether it’s a browser. For instance, major edge environments (e.g., Vercel’s edge runtime, Cloudflare Workers) are not Node-compatible.

Reply:
I wholeheartedly agree.

Reply:
I think the major blocker is solved with: 712127d

Reply:
Looks great to migrate to ts.System